docs/index.md

```diff
@@ -24,6 +26,15 @@ resource "logdna_view" "http500" {
 }
 ```
 
+## Pre-requirements and considerations
+Before configuring your environment to use Terraform for creating LogDNA objects, review the following topics:
+- Verify that Terraform is installed. The minimum supported version is 0.12.0.
+- Have the service key for your Organization available. To obtain the service key fo your LogDNA Organization, go to the LogDNA dashboard and navigate to Settings > Organization > API Keys.
+- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests per minute, therefore when using the LogDNA provider in Terraform there is a limit 50 resource update operations a minute
```

Comment:
This should be support 50 requests at a time from terraform provider rather than per minute.